### PR TITLE
Podman volumes

### PR DESCRIPTION
**Pull request by n0rig**

### CODE REVIEW

**Comment by rhatdan** [comment]
Please squash your commits into a single commit.  Looks like you have included a merge commit.

Please add more test to explain the commit.

Should we have options for sshfs versus 9p?

File: docs/source/markdown/podman-machine-init.1.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ Defaults to `testing`.
 
 Memory (in MB).
 
+#### **--volume**, **-v**=*source:target*
+
+Mounts a volume from `source` to `target`.
```
Comment (rhatdan) [inline docs/source/markdown/podman-machine-init.1.md:52]: This should be a much richer explanation.  
Something like:
```
The source volume will become available to local podman commands.  If you mounted your home directory into the podman machine. Then `podman run  -v /home/dwalsh:/home/dwalsh ...` commands will be able to mount content from these machine directories into containers.
```

**Comment by n0rig** [comment]
Hey, I just wanted to reproduce the build issue seen on afbjorklund@ PR and see where I can contribute to his PR. Apologies for the confusion as I thought this was a draft PR.

File: docs/source/markdown/podman-machine-init.1.md (hunk 1 above)
Comment (afbjorklund) [reply docs/source/markdown/podman-machine-init.1.md:52]: You might want to comment on PR #11454 instead, where this commit was taken from

**Comment by afbjorklund** [comment]
> Should we have options for sshfs versus 9p?

Only 9p (and smb) are included with qemu, so using reverse sshfs would need an external implementation like in [sshocker](https://github.com/AkihiroSuda/sshocker)

The volume flag should be usable with other implementations (thus the "type"), but the mounts are specific to virtfs right now

But please continue the discussion on #8016

**Pull request closed without merging**